caput/mpiarray.py

```diff
@@ -89,6 +92,100 @@
             print(ri, arr3)
         mpiutil.barrier()
 
+Direct Slicing
+==============
+
+The :class: `MPIArray` supports direct slicing, with `__getitem__`.
```

Comment:
```suggestion
The class `MPIArray` supports direct slicing, with `MPIArray.__getitem__`.
```
I'm sort of guessing at your intent here, but i) I don't think sphinx will like the space between the class and the name, and ii) you usually don't need the type specifiers for things within the same file.

Reply:
Also I think this line might confuse some of our less Python-savvy team members into thinking they need to call `__getitem__` directly. Maybe just say it supports slicing using `[...]` as usual and that it's implemented by `__getitem__`.